src/tree/contexts/ContextTreeItem.ts

```diff
@@ -60,6 +60,6 @@ export class ContextTreeItem extends AzExtTreeItem {
     }
 
     public async use(context: IActionContext): Promise<void> {
-        return dockerContextManager.use(this.name);
+        await dockerContextManager.use(this.name);
```

Comment:
The original code is fine, It is only if you mix async code and exception handling where you might get in trouble without `await`.